ops/op2/signature.rs

```diff
@@ -696,6 +699,8 @@ pub enum AttributeModifier {
   Bigint,
   /// #[number], for u64/usize/i64/isize indicating value is a Number
   Number,
+  /// #[memory]
```

Comment:
Bikeshedding a bit, might be better to use `#[wasmmemory]` so it's more obvious to a reader.